sdk/src/identity/identity_assertion/assertion.rs

```diff
@@ -290,6 +289,10 @@ impl IdentityAssertion {
         partial_claim: &PartialClaim,
         status_tracker: &mut StatusTracker,
     ) -> Result<serde_json::Value, ValidationError<String>> {
+        let settings = crate::settings::get_settings().unwrap_or_default();
+        // TO DO (https://github.com/contentauth/c2pa-rs/issues/1454):
+        // Add a Settings argument here?
```

Comment:
TODO: internal

Reply:
I think here we should only pass in a `Trust` struct. Maybe we should also rename it to `TrustSettings`? However, I defer this work to a separate PR. Ideally we don't change public APIs in this one.